gui-libs/display-manager-init/files/startDM

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License, v2
+
+# We need to source /etc/profile for stuff like $LANG to work
+# bug #10190.
+. /etc/profile
+
+. /lib/rc/sh/functions.sh
```

Comment:
This file belongs to openrc, and only below you check if openrc is used. It will fail if no openrc is installed, you might want to change it.

Reply:
done
moved to using `/lib/gentoo/functions.sh` for having `eerror` as needed for exiting non openrc and then resourced this file.